documentation/using_plugins.md

```diff
@@ -265,78 +265,7 @@ google_compute_instance.app.1:
 ```
 
 ## AWS Inventory plugin
-
-The AWS Inventory plugin supports looking up running AWS EC2 instances. It supports several fields:
```

Comment:
The AWS inventory plugin looks up and runs AWS EC2 instances.

Reply:
It looks up currently running instances, it doesn't not run them. I think this could be "The AWS Inventory plugin looks up running AWS EC2 instances."

Reply:
Oh, ha, this is on the other page anyway. Woops!